cmd/fossa/update.go

```diff
@@ -0,0 +1,73 @@
+package main
+
+import (
+	"errors"
+	"fmt"
+
+	"github.com/blang/semver"
+	logging "github.com/op/go-logging"
+	"github.com/rhysd/go-github-selfupdate/selfupdate"
+	"github.com/urfave/cli"
+)
+
+var updateLogger = logging.MustGetLogger("update")
+var updateEndpoint = "fossas/fossa-cli"
+
+func getSemver() string {
+	if version[0] != 'v' {
+		return ""
+	}
+	return version[1:]
+}
+
+func checkUpdate() (bool, error) {
+	latest, found, err := selfupdate.DetectLatest(updateEndpoint)
+	if err != nil {
+		return false, err // unable to update due to GH error
```

Comment:
You should annotate this error with `fmt.Errorf`. For example, `return false, fmt.Errorf("could not detect latest comment: %s", err.Error())`.